### PR TITLE
Upgrade to V38

### PR DESCRIPTION
**Pull request by xbiansf**
Upgrade the data loader to V38 with the latest V38.*  WSC and partner API.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,29 +17,29 @@ The build will generate a windows installer exe in the target directory when exe
 
 To run the Data Loader GUI, run the command
 
-    java -jar target/dataloader-34.0-uber.jar
+    java -jar target/dataloader-38.0-uber.jar
```
Comment (jjangsfdc) [inline README.md]: Nice! you even updated the readme file :)

File: pom.xml [hunk 2]
```diff
@@ -13,8 +13,10 @@
   </organization>
 
   <properties>
-    <force.version>37.0.3</force.version>
-    <build.year>2015</build.year>
+    <force.wsc.version>38.0.3</force.wsc.version>
+    <force.partner.api.version>38.0.0</force.partner.api.version>
```
Comment (jjangsfdc) [inline pom.xml]: Are we good with two versions not in sync? 38.0.3 and 38.0.0?

Comment (apoorvashenoy) [inline pom.xml]: Same question.

Comment (xbiansf) [inline pom.xml]: Yes. We are good.

**Pull request merged**